CMakeLists.txt

```diff
@@ -11,11 +11,11 @@ SET( SYSCONFDIR /etc)
 SET( SYSTEMD_DIR /usr/lib/systemd/system)
 
 if (${PYTHON_DESIRED} STREQUAL "2")
-	FIND_PACKAGE (PythonInterp REQUIRED)
```

Comment:
why here is no PythonLibs? shouldn't we add that too?

Reply:
We probably do not need it at all. This is what in build.log used to be before this change:
-- Could NOT find PythonLibs (missing: PYTHON_LIBRARIES PYTHON_INCLUDE_DIRS) (Required is at least version "3.0")